README.md

```diff
@@ -26,6 +26,8 @@ gems:
 
 ## Configuration
 
+The configuration file is `_config.yml`, the same as Jekyll config file.
+
```

Comment:
@envygeeks how do you recommend me describing the configuration file is the same as jekyll?